imports/server/player.lua

```diff
@@ -32,8 +32,24 @@ function OxPlayer:getState()
     return Player(self.source).state
 end
 
-function OxPlayer:getCoords()
-    return GetEntityCoords(self.ped)
+function OxPlayer:getCoords(heading)
+    local coords = GetEntityCoords(self.ped)
+    if heading == true then
```

Comment:
```suggestion
    if heading then
```

No reason to do a specific true/false check just check if heading has a value

Reply:
Yes, there is a reason. Since `heading` is a boolean, if I use just `if (heading)`, it's going to return true even if I'm calling the function with a number or a string